project/advanced/feedback.rst

```diff
@@ -1,108 +1,63 @@
-======================================
-How to gather feedback from customers?
-======================================
+==============================
+Gather Feedback from Customers
+==============================
 
-Overview
-========
+Finding out what your clients think about the experience they have with your company, can inspire
+you to have insights on how to improve your product/service, adjusting your business to fit
+their needs. It shows that you value their opinions, and provides a reliable source of information
+to other consumers.
```

Comment:
```suggestion
Finding out what your clients think about the experience they have with your company can inspire
you to have insights on how to improve your product/service, adjusting your business to fit
their needs. It shows that you value their opinions and provides a reliable source of information
to other consumers.
```

Reply:
This wasn't resolved. You didn't accept his suggestion as far as I can see.

Reply:
@Lau-Odoo Tu peux recheck stp, de ce que je vois elle a cancel tes suggestions qui était juste.

Reply:
Sure. I'll accept it, even though I didn't in the beginning because I did not agree.

Reply:
As I said, never discard without explaining why it's frustrating for the reviewer. In this case, he was right, the sentence didn't make sense with the comma.

Reply:
Yes, ok. For minor suggestions, I didn't believe an explanation was needed. I apologize. Will always do it from now on. Thank you!